File: foundation/foundation exams/exam foundation 3 month/ex_1.py
import random
def check(k):
	cnt=0
	for x in range(1,k+1):
		if k%x==0:
			cnt+=1
	if cnt==2:
		return True
	else:
		return False
def alls(ms):
	j=0
	while len(ms)>j:
		if check(ms[j])==True:
			m=ms.pop(j)
		else:
			j+=1
	return ms


k=int(input("K ni kiriting: "))
arr=list()
for x in range(k):
	arr.append(random.randint(1,100))
print(arr)
ms=alls(arr)
print(ms)

File: foundation/foundation 2 month/p_exam/11-10-24/v1/ex_1.c
#include <stdio.h>
int check(int a,int b)
{
	return a*b;
}
int main()
{
	int x,y;
	printf("X va Y ni kiriting: ");
	scanf("%d%d",&x,&y);
	printf("Yigindi: %d",check(x,y));
}
